meshtastic/interfaces/ble/client.py

```
@@ -0,0 +1,514 @@
+
+        Returns
+        -------
+            The backend pairing result (typically `None` for bleak >= 2.1.1).
+
+        Raises
+        ------
+            BLEError: If the BLE client is not initialized or if the pairing operation fails.
+
+        """
+        if self.bleak_client is None:
+            raise self.BLEError("Cannot pair: BLE client not initialized")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)